### PR TITLE
Support additional eslint argument for pass-through

### PR DESCRIPTION
**Pull request by cmckendry**
Since I'm using this now, there's a couple of extra arguments that would make it easier to support my normal workflow

### CODE REVIEW

File: lib/configHelpers.js [hunk 1]
```diff
@@ -41,6 +41,10 @@ var configHelpers = module.exports = {
             args.push('--quiet');
         }
 
+        if (opt.options.format) {
+            args.push('--format=' + opt.options.format);
```
Comment (hellatan) [inline lib/configHelpers.js]: how does it format things?

Comment (cmckendry) [inline lib/configHelpers.js]: eslint supports:

- checkstyle
- compact
- html
- jslint-xml
- json
- junit
- stylish (the default)
- table
- tap
- unix
- visualstudio

Personally, I'll mostly be using "compact" but they should all work.

File: index.js [hunk 2]
```diff
@@ -7,12 +8,18 @@ var opt = require('node-getopt').create([
...
+    ['h', 'help', 'display this help'],
+    ['f', 'format=format', 'set output format'],
+    ['v', 'version', 'display eslint version info']
 ])
 .bindHelp()
 .parseSystem();
 
-if (opt.options.git) {
+if (opt.options.version) {
+    var esLintVer = spawnSync('eslint', ['-v']);
```
Comment (hellatan) [inline index.js]: can you convert all of these `var` refs to `const`?

**Comment by hellatan** [comment]
published:

> daletan:~/projects/dibslint (master)
$ npm publish
\+ dibslint@2.0.3

**Pull request merged**